### ISSUE TITLE
Feb 18th Meeting

### ISSUE THREAD

**Issue by wrightaprilm** [macrojc/macrojc#6]
Hi all-

Did we pick a paper for the Feb 18th meeting? Was it this one? http://onlinelibrary.wiley.com/doi/10.1111/evo.12614/abstract

If so, when I create the When is Good poll, do we want to expand beyond Weds, or keep it on Wednesday (ie; search for a better time + day or just for a better time)? Everyone feel free to chime in, especially those of you who've not made it in the past due to scheduling issues.

**Comment by josephwb** [comment]
Inferring :skull: from :deciduous_tree:? I'm in.

**Comment by fmichonneau** [comment]
I think @nhcooper123 already included a whenisgood link to the email she sent out to the google group in her last message.

**Comment by dwbapst** [comment]
This:

http://whenisgood.net/zyhkh4z

But I think we should do a new one for every meeting.

-Dave

On Mon, Feb 9, 2015 at 9:55 AM, Francois Michonneau <
notifications@github.com> wrote:

> I think @nhcooper123 <https://github.com/nhcooper123> already included a
> whenisgood link to the email she sent out to the google group in her last
> message.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/macrojc/macrojc/issues/6#issuecomment-73544709>.
>



-- 
David W. Bapst, PhD
Adjunct Asst. Professor, Geology and Geol. Eng.
South Dakota School of Mines and Technology
501 E. St. Joseph
Rapid City, SD 57701

http://webpages.sdsmt.edu/~dbapst/
http://cran.r-project.org/web/packages/paleotree/index.html

**Comment by nhcooper123** [comment]
Hi April 

Looks like we can condense down to Tuesday, Wednesday and Thursday 12 noon til 5pm EST for polling purposes as this seems to cover the most slots people are able to make. 

Natalie

**Comment by wrightaprilm** [comment]
Cool! I'm pretty hit-or-miss right now - I turn in my thesis in five weeks. Pick times that are good for you, and I'll be there or I won't. I'm happy to be pollmaster, or someone else can.

**Comment by nhcooper123** [comment]
Sorry I dropped the ball a bit with this one. Are we meeting tomorrow? I'm up to my eyeballs at work so I can make it but won't have time to read the paper (I have 6 undergrad theses to read instead, lucky me).

**Comment by dwbapst** [comment]
I'm good with meeting tomorrow.
-Dave

On Tue, Feb 17, 2015 at 1:46 AM, Natalie Cooper <notifications@github.com>
wrote:

> Sorry I dropped the ball a bit with this one. Are we meeting tomorrow? I'm
> up to my eyeballs at work so I can make it but won't have time to read the
> paper (I have 6 undergrad theses to read instead, lucky me).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/macrojc/macrojc/issues/6#issuecomment-74634574>.
>



-- 
David W. Bapst, PhD
Adjunct Asst. Professor, Geology and Geol. Eng.
South Dakota School of Mines and Technology
501 E. St. Joseph
Rapid City, SD 57701

http://webpages.sdsmt.edu/~dbapst/
http://cran.r-project.org/web/packages/paleotree/index.html

**Comment by richfitz** [comment]
It looks like I'm not going to get time to read the paper either.  I did want to read this one too. :cry:

**Comment by bomeara** [comment]
tl;dr: Rabosky 2010: just say no to estimating extinction on molecular
trees. Beaulieu and O'Meara: even with some model violations (varying,
heritable speciation rate) extinction rate can be estimated from such
trees. If the trees are pretty large. And if you like imprecise estimates.
With bias. But it's not nearly as dire as it seemed in 2010.

It's not saying, "go forth and write Nature papers on extinction rates in
your clade of 15 taxa" but rather "stop telling us we absolutely can't
develop methods that use extinction rate".

At least that's the message we hope gets across.

Best,
Brian


_______________________________________
Brian O'Meara
Assistant Professor
Dept. of Ecology & Evolutionary Biology
U. of Tennessee, Knoxville
http://www.brianomeara.info

Postdoc collaborators wanted: http://nimbios.org/postdocs/
Calendar: http://www.brianomeara.info/calendars/omeara

**Comment by josephwb** [comment]
How many people are planning on attending this week? With @richfitz, @wrightaprilm, and @nhcooper123 out, is there insufficient mass? Seems like a paper a lot of people want to read.

**Comment by TGuillerme** [comment]
I'll be there as well!

**Comment by dwbapst** [comment]
Given that a number of people who have significant interest in the paper
seem to be out, I wouldn't mind delaying 1 or 2 weeks, if those people
could attend then.

On Wed, Feb 18, 2015 at 5:23 AM, Joseph W. Brown <notifications@github.com>
wrote:

> How many people are planning on attending this week? With @richfitz
> <https://github.com/richfitz>, @wrightaprilm
> <https://github.com/wrightaprilm>, and @nhcooper123
> <https://github.com/nhcooper123> out, is there insufficient mass? Seems
> like a paper a lot of people want to read.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/macrojc/macrojc/issues/6#issuecomment-74855754>.
>



-- 
David W. Bapst, PhD
Adjunct Asst. Professor, Geology and Geol. Eng.
South Dakota School of Mines and Technology
501 E. St. Joseph
Rapid City, SD 57701

http://webpages.sdsmt.edu/~dbapst/
http://cran.r-project.org/web/packages/paleotree/index.html

**Comment by josephwb** [comment]
Seconded. :boom:

**Comment by TGuillerme** [comment]
I wouldn't mind delaying either but I'll rather stick to the two weeks scheme (March 4th)?
